subsys/bluetooth/mesh/gen_loc_srv.c

```diff
@@ -259,6 +259,24 @@ const struct bt_mesh_model_cb _bt_mesh_loc_srv_cb = {
 	.reset = bt_mesh_loc_srv_reset,
 };
 
+static int bt_mesh_loc_setup_srv_init(struct bt_mesh_model *model)
+{
+	struct bt_mesh_loc_srv *srv = model->user_data;
+#if IS_ENABLED(CONFIG_BT_MESH_COMP_PAGE_1)
```

Comment:
Please avoid using `IS_ENABLED` with `#if`:
https://github.com/zephyrproject-rtos/zephyr/blob/e6197fed5fc9001023fb3d90ec41cbee51594c0c/include/zephyr/sys/util_macro.h#L117-L118

Reply:
Right, so would be better to use `#if (is_defined(CONFIG_BT_MESH_COMP_PAGE_1) && CONFIG_BT_MESH_COMP_PAGE_1==1))`?

Reply:
`#if defined(CONFIG_BT_MESH_COMP_PAGE_1)` should work.

Reply:
What about
```
if (IS_ENABLED(CONFIG_BT_MESH_COMP_PAGE_1)) {
...
}
```
? Always thought it was the cleaner option.

Reply:
Should these be changed as well to avoid using `IS_ENABLED`? https://github.com/nrfconnect/sdk-nrf/blob/baf6159e6140031d8bc3085b3ccc65a529cf9e13/subsys/bluetooth/mesh/light_sat_srv.c#L389

Reply:
I went for the "#if defined()" approach, just changing those related to CDP1.

Reply:
> Should these be changed as well to avoid using `IS_ENABLED`?
> 
> https://github.com/nrfconnect/sdk-nrf/blob/baf6159e6140031d8bc3085b3ccc65a529cf9e13/subsys/bluetooth/mesh/light_sat_srv.c#L389

If you want, you can fix this. In a separate commit or separate PR.